runtime/compute/ARMComputeEx/arm_compute/core/CL/CLKernelLibraryEx.h

```diff
@@ -107,7 +107,7 @@ class CLKernelLibraryEx
    * @param[in] kernel_path Path of the directory from which kernel sources are loaded.
    * @return N/A
    */
-  void set_kernel_path(const std::string &kernel_path) { _kernel_path = kernel_path; };
+  void set_kernel_path(std::string_view kernel_path) { _kernel_path = kernel_path; };
```

Comment:
You can use `const std::string_view` as a parameter. However, since `std::string_view` is a lightweight, non-owning value type that is typically passed by value, adding const doesn't provide extra safety or performance benefits in most cases. It simply prevents modifications to the local copy of the view inside the function, which is often unnecessary.